pkg/xds/generator/secrets/generator.go

```diff
@@ -113,13 +119,13 @@ func (g Generator) Generate(
 	}
 
 	if usedIdentity || len(usedCas) > 0 {
```

Comment:
Can you correct the variable names to be `usedCAs` I struggled to understand this code because I was misreading the variable